spec/monitoring_spec.ts

```diff
@@ -408,6 +408,8 @@ describe('monitoring', () => {
         el(['content-profile-dropdown'], by.buttonText('Simple')).click();
         expect(monitoring.getAllItems().count()).toBe(1);
         expect(monitoring.getTextItemBySlugline(0, 0)).toBe('STORY1 SLUGLINE');
+        expect(browser.isElementPresent(monitoring.getGroupItems(2).get(0))).toBe(false);
```

Comment:
What is this for? The test could be simpler and more readable if you were more explicit and queried items by some identifier like slugline instead of index.

For example, there are `data-test-id="article-item"` and `data-test-id="field--slugline"` which you can use.

Reply:
@tomaskikutis  it's there to check there are no items present in rest of the groups other than the one with slugline `STORY1 SLUGLINE`. There are existing items, which does not contain a slugline, so it's hard to check there presence.

Reply:
Could you use a helper like `monitoring.isGroupEmpty`? It would make your intention clearer.